srcs/runners_manager/runner/RunnerManager.py

```diff
@@ -70,6 +70,9 @@ def update_runners(self, github_runners: list[dict]) -> None:
             self.update_runner(elem)
 
     def create_runner(self) -> None:
+        if 0 < self.vm_type.quantity["max"] <= len(self.runners):
```

Comment:
I don't know if that's a trick I'm not familiar with :D. What is this magic `0 <` ?
```suggestion
        if self.vm_type.quantity["max"] <= len(self.runners):
```

Reply:
Because if max is equal to zero we  don't have a maximum